corehq/apps/hqadmin/utils.py

```diff
@@ -0,0 +1,19 @@
+from dimagi.utils.logging import notify_exception
+from pillowtop.utils import get_all_pillows
+
+from .models import PillowCheckpointSeqStore
+
+
+def pillow_seq_store():
+    for pillow in get_all_pillows():
+        checkpoint = pillow.get_checkpoint()
+        store, created = PillowCheckpointSeqStore.objects.get_or_create(checkpoint_id=checkpoint['_id'])
+        if not created and checkpoint['seq'] < store.checkpoint_id:
+            notify_exception(
```

Comment:
I think this is still a good idea, but maybe split off the first part and parse as int (there's probably a function that checks whether it's a string or int, and does the proper thing to return an int), and then check if it's behind by more than 10000 or something.